clang/lib/CodeGen/CGCall.cpp

```diff
@@ -5686,6 +5686,10 @@ RValue CodeGenFunction::EmitCall(const CGFunctionInfo &CallInfo,
   if (!CI->getType()->isVoidTy())
     CI->setName("call");
 
+  if (getTarget().getTriple().isSPIRVLogical() &&
+      CI->getCalledFunction()->isConvergent())
```

Comment:
Can you drop the getCalledFunction part?

Reply:
Yes, thanks